samples/spring-security-xsuaa-usage/pom.xml

```diff
@@ -81,6 +81,14 @@
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-war-plugin</artifactId>
```

Comment:
is war plugin really necessary for spring projects? spring mvn plugin is already there are you sure that we need two build plugins?